crates/moss-theme/src/schema.rs

```diff
@@ -0,0 +1,130 @@
+use serde_json::{json, Value};
```

Comment:
I think the file header should include a note that the file was generated automatically and should not be edited manually